_includes/schema-iteration-list.md

```diff
@@ -5,19 +5,20 @@
 {%- assign segments = file.path | split: "/" -%}
 {%- if segments[1] == include.specification -%}
 {%- if segments[2] != last_version -%}
-{%- assign last_version = segments[2] %}
+{% assign last_version = segments[2] %}
+{% assign last_kind = "" %}
```

Comment:
In this file we're using a mix of [whitespace-trimming](https://shopify.github.io/liquid/basics/whitespace/) and whitespace-preserving tags.

Is there a specific reason these are changed to whitespace-preserving?
If not, consider maintaining the whitespace-trimming style.

Reply:
This mix achieves the desired output, and all other variations I tried did not. 

What I haven't tried is all whitespace-trimming with empty lines to explicitly insert whitespace in the output. Feel free to experiment.

Reply:
Whitespace-trimming with empty lines doesn't work - they are trimmed away 🙄

Reply:
Is there a way to run this functionality locally, or does it need to be processed through Github Pages?

Reply:
https://github.com/OAI/OpenAPI-Specification/tree/gh-pages?tab=readme-ov-file#development